### PR TITLE
Removed unused imports and mutability issues.

### PR DESCRIPTION
**Pull request by BenCaunt**

### CODE REVIEW

**Comment by vwu1888** [comment]
lgtm

**Comment by vwu1888** [comment]
> lgtm

disregard dlgtm

**Comment by BenCaunt** [comment]
looks bad to me

File: src/missions/vision.rs
Comment (Bennett-Petzold) [inline src/missions/vision.rs:1]: mut is required when "logging" is enabled, this will break some builds.

Comment (ericliu0000) [reply src/missions/vision.rs:1]: I think this is fixed now; unsure of whether it is the most elegant solution but it builds.

**Comment by BenCaunt** [comment]
lgtqm

**Review by Bennett-Petzold — APPROVED** [review]

**Pull request merged**